app/controllers/authenticate_controller.rb

```diff
@@ -200,7 +200,8 @@ def handle_authentication_error(err)
 
   def log_backtrace(err)
     err.backtrace.each do |line|
-      logger.debug(line)
+      break if line.include?("2.5.0/gems")
+      logger.info(line)
```

Comment:
It should be possible to get the full output in the debug logs without a code change, so something like (pseudo-code):
```
line.include?("2.5.0/gems") ? logger.debug(line) : logger.info(line)
```

Reply:
I agree with this but wouldn't you just wrap Oren's new `break` statement in something like (pseudocode) `unless LOGLEVEL == 'debug'`?

Reply:
> I agree with this but wouldn't you just wrap Oren's new break statement in something like (pseudocode) unless LOGLEVEL == 'debug'?

I _think_ that would probably functionality accomplish the same outcome, and that was actually what I was initially going to comment here. However, I thought it was it was awkward to read:

```
if ... DEBUG ...
   logger.info
```

When that's was `logger.debug` is for. I thought the suggestion above was more clear about the distinction of what was DEBUG level output and what was INFO.

Reply:
you are both correct. please see the updated code.

Do you think this is good enough for us to push? if so i will create an issue and a changelog entry